be/app/Http/Controllers/AuthController.php

```diff
@@ -31,14 +31,13 @@ public function register(Request $request)
 
         $token = $user->createToken('permission')->plainTextToken;
 
-        $response = [
+        return response([
             'status' => 200,
```

Comment:
- ahh i missed out this one. Please remove this one, specify your status code in `line:40` (apply to all related).